### ISSUE TITLE
Blank display on reddit.com

### ISSUE THREAD

**Issue by CEX68** [sapjax/word-hunter#40]
<img width="854" alt="1" src="https://github.com/sapjax/word-hunter/assets/115330187/19ac728f-440a-4463-a0d2-17793ba4c515">
on https://www.reddit.com/r/EnglishLearning/comments/1b7m2kf/why_does_the_possessive_pronoun_for_it_not_exist/

Version: 1.1.8
It's normal on other websites.

**Comment by sapjax** [comment]
> https://www.reddit.com/r/EnglishLearning/comments/1b7m2kf/why_does_the_possessive_pronoun_for_it_not_exist/

The blank whitespace is the Chinese translation,
And the CSS content of Reddit site hides any custom element.

```css
:not(:defined) {
    visibility: hidden;
}
```

Currentlly you can add custom CSS to the [Stylus](https://chromewebstore.google.com/detail/Stylus/clngdbkpkpeebahjckkjfobafhncgmne) to make it visible:
```css
w-mark-t {
    visibility: visible !important;
}
```

I'll fix it in the next version.

**Comment by sapjax** [comment]
fixed in v1.1.9
